sensor_msgs/PointCloud2 stitched_cloud
---
std_msgs/Float64MultiArray alignment_transformation